datadog_lambda/metric.py

```diff
@@ -52,13 +55,15 @@ def lambda_metric(metric_name, value, timestamp=None, tags=None):
     """
     tags = _tag_dd_lambda_layer(tags)
     if os.environ.get('DD_FLUSH_TO_LOG', '').lower() == 'true':
+        logger.debug('Sending metric to Datadog via log forwarder')
         print(json.dumps({
             'm': metric_name,
             'v': value,
             'e': timestamp or int(time.time()),
             't': tags
         }))
     else:
+        logger.debug('Sending metric to Datadog via lambda layer')
```

Comment:
Probably would be a good idea to print the metric name in this message. Also, if we could check that the API_KEY is set, or maybe print its length. I feel like people failing to set the API key correctly is the most common screw up, and something that lets us catch that would be helpful.